pages/index.tsx

```diff
@@ -25,8 +26,16 @@ type Props = {
 };
 
 export default function Home({ deviceType }: Props) {
+  const [url, setURL] = useState(BASE_URL);
+  const coordinates: CoordinatesURLParameters | undefined = useCoordinates();
+
+  useEffect(() => {
+    const urlParams = new URLSearchParams(coordinates as any);
+    setURL(BASE_URL + "?" + urlParams.toString());
+  }, [coordinates]);
+
   const { error, isLoading } = useSWR<MarkerData[] | undefined>(
-    BASE_URL,
+    url,
```

Comment:
Demek istedigim soyle bir seydi, kontrol etmemiz gereken `coordinates` mi tam emin degilim. 

```suggestion
    coordinates ?  url : null,
```

Reply:
Anladim 🙏 Hmm, benzer bir sekilde çözebiliriz. Insanlar pr'a bakana kadar bir bakayım.